src/docs/guides/csp-nonce.md

```diff
@@ -79,11 +78,11 @@ This is an example of consuming the `dist-custom-elements` output in an Angular
 // main.ts
 
 import { defineCustomElements, setNonce } from 'my-lib/dist/components';
-// We'll assume we have `autoDefineCustomElements` enabled
+// Assume `autoDefineCustomElements` is enabled enabled
```

Comment:
How do we want to handle this once `autoDefineCustomElements` goes away in v3.0.0?

Reply:
Once this lands and we rebase v3, I can put up a small PR to change this comment to the export behavior instead